packages/common/src/types/filter.ts

```diff
@@ -66,12 +66,18 @@ export type DashboardFieldTarget = {
     tableName: string;
 };
 
+export type TileFieldTargetOverride = DashboardFieldTarget & {
+    tileUuid: string;
+};
+
 export type DashboardFilterRule<
     O = FilterOperator,
     T = DashboardFieldTarget,
     V = any,
     S = any,
-> = FilterRule<O, T, V, S>;
+> = FilterRule<O, T, V, S> & {
+    tileTargetOverride?: TileFieldTargetOverride[];
```

Comment:
could this be a map `Record<string,DashboardFieldTarget>` since it doesn't make sense to have the same tile uuid twice? It would also remove the additional type

Reply:
updated